Automattic/wp-calypso
# Remove Reader Lists Flux store

Removes the last uses of the Reader Lists Flux store and actions, and also the Flux store itself.

Creating a new list is not currently supported, so it's safe to remove the code that enabled that.

Part of https://github.com/Automattic/wp-calypso/issues/10984.

### To test

Using an account that already has Reader lists set up, make sure viewing your lists from the sidebar works as expected.
